src/components/Thanks/MyKiva/ThanksBadges.vue

```diff
@@ -183,9 +199,9 @@ const props = defineProps({
 		type: Object,
 		default: () => ({}),
 	},
-	selectedLoan: {
-		type: Object,
-		default: () => ({})
+	numberOfBadgesEarned: {
```

Comment:
Instead of a new prop, you can create a new `computed` that returns the length of `props.badgesAchieved`